osquery/tables/system/darwin/screenlock.mm

```diff
@@ -0,0 +1,90 @@
+/**
+ *  Copyright (c) 2014-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed in accordance with the terms specified in
+ *  the LICENSE file found in the root directory of this source tree.
+ */
+
+#import <Foundation/Foundation.h>
+#include <limits.h>
+#include <osquery/logger.h>
+#include <osquery/tables.h>
+#include <stdbool.h>
+
+namespace osquery {
+namespace tables {
+
+void logUnsupportedError() {
```

Comment:
I think this helper could take a string argument and log a 1-liner “... not supported ...: reason”

Reply:
Totally agree, will change right away.